website/static/js/search.js

```diff
@@ -146,14 +209,43 @@ var ViewModel = function(params) {
         };
     });
 
-    self.fullQuery = ko.pureComputed(function() {
+    self.filters = function(){
+        var licenses = self.selectedLicenses();
+        if (licenses.length) {
+            var filters = {
+                terms: {
+                    'license.name': licenses
+                }
+            };     
+            if (licenses.filter(function(l) {
+                return l.name === DEFAULT_LICENSE.name;
```

Comment:
This is brittle; what if the default license's name changes? Can we use `id` instead?